pkgs/by-name/sk/skimpdf/package.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, lib
+, undmg
+, fetchurl
+,
+}:
+stdenv.mkDerivation rec {
+  pname = "Skim-${version}";
+  version = "1.6.21";
+
+  src = fetchurl {
+    name = "Skim-${version}.dmg";
+    url = "mirror://sourceforge/project/skim-app/Skim/Skim-${version}/Skim-${version}.dmg";
+    sha256 = "0SeS41ExCZfGmjiK2KvvdFpP7IVzm7xHKCaKmEUYQAY=";
+  };
+
+  nativeBuildInputs = [ undmg ];
+
+  sourceRoot = ".";
+
+  installPhase = ''
+    mkdir -p $out/Applications
+    cp -R *.app $out/Applications
+  '';
+
+  meta = with lib; {
+    description = "Skim is a PDF reader and note-taker for OS X";
+    homepage = "https://skim-app.sourceforge.io/";
+    license = licenses.bsd0;
+    mainProgram = "Skim.app";
```

Comment:
If there is one `mainProgram` you shouldn't copy multiple binaries to $out (`cp -R *.app $out/Applications` in installPhase)

If there are many executables, this `meta.mainProgram` should not be defined

https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes

Reply:
Still relevant

Reply:
There is only Skim.app so the mainProgram tag should be fine

Reply:
Oh wait sorry, for some reason my changes to the installPhase did not get synced, fixing it right now

Reply:
Has been done